x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/geo/StDistance.java

```diff
@@ -83,4 +72,22 @@ protected Pipe makePipe() {
     protected String scriptMethodName() {
         return "stDistance";
     }
+
+    public static class StDistanceFunction implements PredicateBiFunction<Object, Object, Double> {
```

Comment:
I would define it in as a normal class in a separate file instead.

Reply:
:+1: